docs/userguide/format_parse/messages/try-mf2.md

```diff
@@ -0,0 +1,366 @@
+        * "Debugging" set "Environment" to \
+          `PATH=$(IcuDistro)/bin;%PATH%`
+
+1. Update the default source file to use some ICU functionality
+
+    * Select the "View" -- "Solution Explorer" menu
+    * In the left-side tree "HelloMF2" / "Source Files" open `HelloMF2.cpp`
+    * Copy-paste the code from the `hello_mf2.cpp` (in this document) and paste it in
+      `HelloMF2.cpp`, replacing all the existing content (select all + paste).
+
+1. At this point you should be able to build and run the application, debug it, etc.
+
+1. When run, it will output
+    ```
+    "Hello John, today is January 13, 2025!"
+    ```
+
+### From Visual Studio with minimal work
+
+These instructions will use a release version of ICU4C (tested with 76.1) and
+a "Hello ICU world" project already created.
+They provide minimal effort that only requires downloading and opening in Visual Studio
+before using.
+
+> :point_right: **Note**: the MessageFormat 2 implementation in a previously release version may be behind the spec in the first few releases of ICU after the MF2.0 spec was declared 1.0 in CLDR 46.1,
+which occurred between ICU 76 and ICU 77. The difference between latest MF2.0 and the spec version supported in ICU may be minimal.
+
+
+1. Download the Visual Studio artifacts from the
+[official release of ICU 76.1](https://github.com/unicode-org/icu/releases/tag/release-76-1):
+    * [icu4c-76_1-Win32-MSVC2022.zip](https://github.com/unicode-org/icu/releases/download/release-76-1/icu4c-76_1-Win32-MSVC2022.zip)
+    * [icu4c-76_1-Win64-MSVC2022.zip](https://github.com/unicode-org/icu/releases/download/release-76-1/icu4c-76_1-Win64-MSVC2022.zip)
+
+1. Download the "Hello ICU / MF2 World" project [HelloMF2.zip](HelloMF2.zip).
+
+1. Unzip the files you just downloaded and merge the content of the two ICU folders.
+The "hello world" project to be a sibling to the icu folder.
+
+1. The folder tree structure should look like this
+    
+    ```
+    someFolderOfYourChoice\
+      +- HelloMF2\
+      | \- HelloMF2\
+      \- icu\
+        \- icu4c\
+          +- bin\
+          +- bin64\
+          +- include\
+          | \- unicode\
+          +- lib\
+          \- lib64\
+    ```
+
+1. Open the `HelloMF2.sln` solution in Visual Studio and you are ready to go.
+
+
+## Java
+
+We will assume that if you are interested in testing a pre-release Java library
+you already have (or know how to install) a JDK, Apache Maven, git,
+know how to create a project in your favorite IDE, and so on.
+
+### What you need
+
+* JDK, any version between 8 and 23
+* Apache Maven
+
+### Instructions
+
+1. Create a new Maven project
+    ```sh
+    mvn archetype:generate -DgroupId=com.mycompany.app -DartifactId=hello_icu_mf2 -DarchetypeArtifactId=maven-archetype-quickstart -DarchetypeVersion=1.5 -DinteractiveMode=false
+
+    cd hello_icu_mf2
+    ```
+
+1. Edit the `pom.xml` file
+
+    1. The project created as above uses the Java 17 version.
```

Comment:
This should not be under "Edit the pom.xml file"

Edit ... means "open it in an editor and do ..."